packages/go_router_builder/example/test/all_types_test.dart

```diff
@@ -110,5 +110,15 @@ void main() {
     expect(find.text('UriRoute'), findsOneWidget);
     expect(find.text('Param: https://dart.dev'), findsOneWidget);
     expect(find.text('Query param: https://dart.dev'), findsOneWidget);
+
+    IterableRoute(
+      intListField: <int>[1, 2, 3],
+    ).go(scaffoldState.context);
+    await tester.pumpAndSettle();
+    expect(find.text('IterableRoute'), findsOneWidget);
+    expect(
+        find.text(
+            '/iterable-route?int-list-field=1&int-list-field=2&int-list-field=3'),
```

Comment:
Seems a bit weird to hardcoded the query parameter name, I think this may make it less useful if a company care about the query parameter names. Would it make more sense to fix https://github.com/flutter/flutter/issues/110781 and provide an API that people can set it up for iterable object with minimum code?

Reply:
this seems not addressed yet?